source/_patterns/01-molecules/components/highlight-item.mustache

```diff
@@ -0,0 +1,40 @@
+<li class="highlight-item" data-behaviour="HighlightItem">
```

Comment:
We do not need data-behaviour attribute as we are not assigning any js to this item.